test/e2e/data/e2e_conf.yaml

```diff
@@ -172,7 +172,7 @@ variables:
   OPENSTACK_DNS_NAMESERVERS: "8.8.8.8"
   OPENSTACK_FAILURE_DOMAIN: "testaz1"
   OPENSTACK_FAILURE_DOMAIN_ALT: "testaz2"
-  OPENSTACK_IMAGE_NAME: "ubuntu-2204-kube-v1.30.1"
+  OPENSTACK_IMAGE_NAME: "ubuntu-2204-kube-v1.31.2"
   OPENSTACK_IMAGE_NAME_UPGRADE_FROM: "ubuntu-2204-kube-v1.29.5"
```

Comment:
I think this must be bumped to v1.30 or the upgrade will fail because of attempt to skip over versions